ipatests/test_integration/test_sssd.py

```diff
@@ -403,3 +407,185 @@ def test_trustdomain_disable_disables_subdomain(self):
+
+        ssh_key_file_contents = ''
+        with open(user_ssh_priv_key, 'r') as infile:
+            ssh_key_file_contents = infile.read()
+        master.put_file_contents('/tmp/user_ssh_priv_key',
+                                 ssh_key_file_contents)
+        master.run_command('chmod 600 /tmp/user_ssh_priv_key')
+
+        # add group parent
+        cmd_output = master.run_command(['ipa', 'group-add', 'parent'])
+        assert 'Added group "parent"' in cmd_output.stdout_text
+
+        # add group child1
+        cmd_output = master.run_command(['ipa', 'group-add', 'child1'])
+        assert 'Added group "child1"' in cmd_output.stdout_text
+
+        # add group child2
+        cmd_output = master.run_command(['ipa', 'group-add', 'child2'])
+        assert 'Added group "child2"' in cmd_output.stdout_text
+
+        # add group members
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'child1', '--groups', 'parent'])
+        assert 'Group name: child1' in cmd_output.stdout_text
+        assert 'Member groups: parent' in cmd_output.stdout_text
+        assert 'Number of members added 1' in cmd_output.stdout_text
+
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'child2', '--groups', 'child1'])
+        assert 'Group name: child2' in cmd_output.stdout_text
+        assert 'Member groups: child1' in cmd_output.stdout_text
+        assert 'Indirect Member groups: parent' in cmd_output.stdout_text
+
+        # add user to group 'parent'
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'parent', '--users', username])
+        assert 'Group name: parent' in cmd_output.stdout_text
+        assert 'Member users: {}'.format(username) in cmd_output.stdout_text
+        assert 'Member of groups: child1' in cmd_output.stdout_text
+        assert 'Indirect Member of group: child2' in cmd_output.stdout_text
+
+        # clear sssd_cache
+        clear_sssd_cache(master)
+
+        # user lookup
+        cmd_output = master.run_command(['ipa', 'user-show', username])
+        assert 'parent' in cmd_output.stdout_text
+        assert 'ipausers' in cmd_output.stdout_text
+        assert 'child1' in cmd_output.stdout_text
+        assert 'child2' in cmd_output.stdout_text
+
+        clear_sssd_cache(master)
+
+        try:
+            cmd = 'ssh -i {} -q {}@{} groups'.format('/tmp/user_ssh_priv_key',
+                                                     username, client.hostname)
+            cmd_output = master.run_command(cmd)
+            assert '{}'.format(username) in cmd_output.stdout_text
+            assert 'parent' in cmd_output.stdout_text
+            assert 'child1' in cmd_output.stdout_text
+            assert 'child2' in cmd_output.stdout_text
+        except (paramiko.AuthenticationException,
+                paramiko.SSHException) as e:
```

Comment:
or this except block catching paramiko exceptions (we are not using paramiko)

Reply:
Removed it